api_tests/nodes/views/test_node_draft_registration_list.py

```diff
@@ -126,54 +216,37 @@ def test_admin_can_view_draft_list(
         assert data[0]['id'] == draft_registration._id
         assert data[0]['attributes']['registration_metadata'] == {}
 
-    def test_osf_group_with_admin_permissions_can_view(
-            self, app, user, draft_registration, project_public,
-            schema, url_draft_registrations):
-        group_mem = AuthUserFactory()
-        group = OSFGroupFactory(creator=group_mem)
-        project_public.add_osf_group(group, permissions.ADMIN)
-        res = app.get(url_draft_registrations, auth=group_mem.auth, expect_errors=True)
+    def test_node_admin_can_view_draft_list(
+            self, app, user, draft_registration, project_public, user_admin_contrib, schema, url_draft_registrations
+    ):
+        """
+        Project Admins should be implicitly granted READ permissions on Drafts even if not explicitly a contributor.
+        """
+        draft_registration.remove_contributor(user_admin_contrib, Auth(user_admin_contrib), _force=True)
+        assert user_admin_contrib not in draft_registration.contributors.all()
+        assert project_public.has_permission(user_admin_contrib, permissions.ADMIN)
+
+        res = app.get(url_draft_registrations, auth=user_admin_contrib.auth)
         assert res.status_code == 200
         data = res.json['data']
         assert len(data) == 1
-        assert schema._id in data[0]['relationships']['registration_schema']['links']['related']['href']
 
-    def test_cannot_view_draft_list(
-            self, app, user_write_contrib, project_public,
-            user_read_contrib, user_non_contrib,
-            url_draft_registrations, group, group_mem):
-
-        # test_read_only_contributor_cannot_view_draft_list
-        res = app.get(
-            url_draft_registrations,
-            auth=user_read_contrib.auth,
-            expect_errors=True)
-        assert res.status_code == 403
+    def test_read_only_contributor_can_view_draft_list(self, app, user_read_contrib, url_draft_registrations):
+        res = app.get(url_draft_registrations, auth=user_read_contrib.auth)
+        assert res.status_code == 200
 
-    #   test_read_write_contributor_cannot_view_draft_list
-        res = app.get(
-            url_draft_registrations,
-            auth=user_write_contrib.auth,
-            expect_errors=True)
-        assert res.status_code == 403
+    def test_read_write_contributor_can_view_draft_list(self, app, user_write_contrib, url_draft_registrations):
+        res = app.get(url_draft_registrations, auth=user_write_contrib.auth)
+        assert res.status_code == 200
 
-    #   test_logged_in_non_contributor_cannot_view_draft_list
-        res = app.get(
-            url_draft_registrations,
-            auth=user_non_contrib.auth,
-            expect_errors=True)
+    def test_logged_in_non_contributor_cannot_view_draft_list(self, app, user_non_contrib, url_draft_registrations):
```

Comment:
Non-contributor on the project who is a contributor on the Draft should still see the draft in the list. This shows that they will 403 instead (because we're checking `has_permission(user, READ)` instead of `can_view(user)`)